fondant/component.py

```diff
@@ -120,7 +130,7 @@ def _load_or_create_manifest(self) -> Manifest:
         return manifest
 
     @abstractmethod
-    def load(self, args: argparse.Namespace) -> dd.DataFrame:
+    def load(self, **kwargs) -> dd.DataFrame:
```

Comment:
Same comments as fro the `transform` method (my bad should have included this in my PR)